src/js/atomDataTable.js

```diff
@@ -1,33 +1,54 @@
-(function ($) {
-  $.fn.atomDataTable = function (customOptions) {
-    var sortClass = "a-sort";
-    var directionAttr = "data-direction";
-    const direction = ["asc", "desc"];
-    const types = [
-      "string",
-      "number",
-      "Ymd",
-      "m/d/Y",
-      "m-d-y",
-      "d/m/Y",
-      "d-m-y",
-    ];
-
-    var tableid =
+const direction = ["asc", "desc"];
+const types = [
```

Comment:
To avoid clashes with other scripts
```suggestion
const a_directions = ["asc", "desc"];
const a_types = [
```